mono/metadata/image.c

```diff
@@ -503,9 +499,6 @@ load_tables (MonoImage *image)
 		} else {
 			image->tables [table].rows = read32 (rows);
 		}
-		/*if ((sorted_mask & ((guint64) 1 << table)) == 0){
-			g_print ("table %s (0x%02x) is sorted\n", mono_meta_table_name (table), table);
```

Comment:
This removes a comment that contains code that might or might not be useful, might or might not compile and probably hasn't been used for years.